Northstar.CustomServers/mod/scripts/vscripts/sh_utility_all.gnut

```diff
@@ -1532,6 +1532,21 @@ array<string> function GetAvailableTitanRefs( entity player )
 	return availableTitanRefs
 }
 
+void function RecalculateHighestTitanFDLevel( entity player )
+{
+	//Refresh Highest Aegis Titan since we might get all of them back to 1 if players wants
+	int enumCount = PersistenceGetEnumCount( "titanClasses" )
+	int HighestAegis = 0
+	for ( int i = 0; i < enumCount; i++ )
+	{
+		string enumName = PersistenceGetEnumItemNameForIndex( "titanClasses", i )
+		int AegisLevel = FD_TitanGetLevelForXP( enumName, FD_TitanGetXP( player, enumName ) )
+		if ( HighestAegis < AegisLevel )
+			HighestAegis = AegisLevel
+	}
+	player.SetPersistentVar( "fdStats.highestTitanFDLevel", HighestAegis )
```

Comment:
```suggestion
	int enumCount = PersistenceGetEnumCount( "titanClasses" )
	int highestAegis = 0
	for ( int i = 0; i < enumCount; i++ )
	{
		string enumName = PersistenceGetEnumItemNameForIndex( "titanClasses", i )
		int aegisLevel = FD_TitanGetLevelForXP( enumName, FD_TitanGetXP( player, enumName ) )
		if ( highestAegis < aegisLevel )
			HighestAegis = AegisLevel
	}
	player.SetPersistentVar( "fdStats.highestTitanFDLevel", highestAegis )
```
camelCase and removal of irrelevant comment because this used to be part of another function